src/app/common/stores/applications/SsvWeb/Wallet.store.ts

```diff
@@ -183,6 +186,8 @@ class WalletStore extends BaseStore implements Wallet {
       this.ssvStore.clearSettings();
       myAccountStore.clearIntervals();
       this.accountAddress = address;
+      const accountStore: AccountStore = this.getStore('Account');
+      await accountStore.getFeeRecipientAddress(this.accountAddress);
```

Comment:
Instead of making all these calls here one after another in a blocking mode, let's do something like:

```
await Promise.all([
  accountStore.onAccountAddressChange(this.accountAddress),
  this.initializeUserInfo(),
  myAccountStore.getOwnerAddressOperators({}),
])
```

So all 3 of them will happen in parallel and speed of the page loading will be higher.

Also, there should be some mechanism (if not yet), that tries as a fallback few times more with delays between attempts.

Also pay attention to this part: `accountStore.onAccountAddressChange` 
If you need to load fee recipient - you call this inside of `accountStore.onAccountAddressChange` method, but not here
If you don't need (which I'm pretty sure about) - you need to get fee recipient address somewhere on a React component level once when it loads.

Reply:
done